### ISSUE TITLE
Add name to Hamburger menu on all pages

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#125]
From NYU A11y review -

At various responsive break points, the Uploads and Browse All navigation links move into a hamburger menu. This menu does not have an accessible name.

(This means aria-label or other approach which makes text legible to screenreaders.)

**Comment by lhenze** [comment]
Here's a useful article on this subject https://www.ashleysheridan.co.uk/blog/Making+an+Accessible+Hamburger+Menu
